.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Code Question
+about: Ask a question about the code
+title: ''
+labels: 'Type: Question'
+assignees: ''
+
+---
+
+<!--
+Thank you for taking an interest in the OpenZFS codebase.
+
+Please be aware that most questions are preferably asked in the mailing list first.
+This form is primarily meant for asking questions about the code itself.
+
+Please also check our issue tracker before opening a new question.
+Filling out the following template will help other contributors better understand your question.
+-->
+
+### Ask your question!
+
+<!--
+Provide a clear and concise question and please add as much information as possible.
```

Comment:
Sorry, reading this over again the wording here feels a bit contradictory.  It's asking them to be concise yet provide as much information as possible.  How about dropping the "and please add as much information as possible." portion?

Reply:
No problem, thats what feedback is for! :)
But you're right, it seems a bit contradictory.

I've send in a fix and added "Please" in front, because it became a bit... passive-agressive otherwise imho.